upgrade_test.py

```diff
@@ -110,34 +132,51 @@ def rollback_node(self, node):
         orig_ver = result.stdout
-        node.remoter.run('sudo cp ~/scylla.repo-backup /etc/yum.repos.d/scylla.repo')
         # flush all memtables to SSTables
         node.remoter.run('nodetool drain')
         # backup the data
         node.remoter.run('nodetool snapshot')
-        node.remoter.run('sudo systemctl stop scylla-server.service')
-        node.remoter.run('sudo chown root.root /etc/yum.repos.d/scylla.repo')
-        node.remoter.run('sudo chmod 644 /etc/yum.repos.d/scylla.repo')
-        node.remoter.run('sudo yum clean all')
+        node.stop_scylla_server(verify_down=False)
+
+        node.remoter.run('sudo cp /etc/scylla/scylla.yaml-backup /etc/scylla/scylla.yaml')
+        if node.is_rhel_like():
+            node.remoter.run('sudo cp ~/scylla.repo-backup /etc/yum.repos.d/scylla.repo')
+            node.remoter.run('sudo chown root.root /etc/yum.repos.d/scylla.repo')
+            node.remoter.run('sudo chmod 644 /etc/yum.repos.d/scylla.repo')
+            node.remoter.run('sudo yum clean all')
+        else:
+            node.remoter.run('sudo cp ~/scylla.list-backup /etc/apt/sources.list.d/scylla.list')
+            node.remoter.run('sudo chown root.root /etc/apt/sources.list.d/scylla.list')
+            node.remoter.run('sudo chmod 644 /etc/apt/sources.list.d/scylla.list')
+            node.remoter.run('sudo apt-get clean all')
+            node.remoter.run('sudo apt-get update')
 
         if re.findall('\d+.\d+', self.orig_ver)[0] == re.findall('\d+.\d+', self.new_ver)[0]:
             self.upgrade_rollback_mode = 'minor_release'
 
-        if self.upgrade_rollback_mode == 'reinstall':
-            node.remoter.run('sudo yum remove scylla\* -y')
-            node.remoter.run('sudo yum install %s -y' % node.scylla_pkg())
-            node.remoter.run('for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ); do sudo cp -v $conf.autobackup $conf; done')
+        if self.upgrade_rollback_mode == 'reinstall' or not node.is_rhel_like():
+            if node.is_rhel_like():
+                node.remoter.run('sudo yum remove scylla\* -y')
+                node.remoter.run('sudo yum install %s -y' % node.scylla_pkg())
+                node.remoter.run('for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ); do sudo cp -v $conf.autobackup $conf; done')
+            else:
+                node.remoter.run('sudo apt-get remove scylla\* -y')
+                node.remoter.run('sudo apt-get install %s -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated' % node.scylla_pkg())
+                node.remoter.run('for conf in $(cat /var/lib/dpkg/info/scylla-*server.conffiles /var/lib/dpkg/info/scylla-*conf.conffiles /var/lib/dpkg/info/scylla-*jmx.conffiles | grep -v init ); do sudo cp -v $conf.backup $conf; done')
+                node.remoter.run('sudo systemctl daemon-reload')
         elif self.upgrade_rollback_mode == 'minor_release':
```

Comment:
this flow is checked only for rhel like OS-s. Is it intended so?

Reply:
We always use reinstall mode for ubuntu/debian rollback.
So the minor_release and smooth mode are only for RHEL.